data/coalition/lunarium intro.txt

```diff
@@ -1900,7 +1900,7 @@ mission "Lunarium: Quarg Interview"
-			`	Some of the others at the table look to one another, and one of the Kimek who accompanied you all the way from your ship sighs quietly before coming to Lobmab. "Most welcome you are, Lady Lobmab," he says, as he begins pointing to the leaders one by one, starting from your left with a Kimek. "Your host, Chiree is, the head of our local forces and organizer of many of our operations. Fought the Heliarchs since a young age, she has." He continues pointing to them in a clockwise pattern, now to the left of Chiree. "Likewise, been helping us for decades now, Tummug has, in spreading word of our group, and bringing to the people the truth about the Heliarchy. Another prominent Arach, and vital to the task soon to be discussed, Oobat is. A studious linguist, and aspiring diplomat, she is. Already worked together, I believe you and Pyakri have? One of our most senior militia officers, she is." He takes a pause, looking at Lobmab, almost as if waiting for some approval regarding his introductions of the others. She simply nods, and he moves on. "And... Elirom. A prominent Saryd Heliarch Arbiter who defected over a century ago. Been advising us since, he has." He waits for her to say something, before looking back at the table again and pointing to you. "Oh, of course. <first> <last>, the human, helped us greatly who has, and joining us today, who is."`
+			`	Some of the others at the table look to one another, and one of the Kimek who accompanied you all the way from your ship sighs quietly before coming to Lobmab. "Most welcome you are, Lady Lobmab," he says, as he begins pointing to the leaders one by one, starting from your left with a Kimek. "Your host, Chiree is, the head of our local forces and organizer of many of our operations. Fought the Heliarchs since a young age, she has." He continues pointing to them in a clockwise pattern, now to the left of Chiree. "Likewise, been helping us for decades now, Tummug has, in spreading word of our group, and bringing to the people the truth about the Heliarchy. Another prominent Arach, and vital to the task soon to be discussed, Oobat is. A studious linguist, and aspiring diplomat, she is. Already worked together, I believe you and Pyakri have? One of our most senior militia officers, she is." He takes a pause, looking at Lobmab, almost as if waiting for some approval regarding his introductions of the others. She simply nods, and he moves on. "And... elirom. A prominent Saryd Heliarch Arbiter who defected over a century ago. Been advising us since, he has." He waits for her to say something, before looking back at the table again and pointing to you. "Oh, of course. <first> <last>, the human, helped us greatly who has, and joining us today, who is."`
```

Comment:
```suggestion
			`	Some of the others at the table look to one another, and one of the Kimek who accompanied you all the way from your ship sighs quietly before coming to Lobmab. "Most welcome you are, Lady Lobmab," he says, as he begins pointing to the leaders one by one, starting from your left with a Kimek. "Your host, Chiree is, the head of our local forces and organizer of many of our operations. Fought the Heliarchs since a young age, she has." He continues pointing to them in a clockwise pattern, now to the left of Chiree. "Likewise, been helping us for decades now, Tummug has, in spreading word of our group, and bringing to the people the truth about the Heliarchy. Another prominent Arach, and vital to the task soon to be discussed, Oobat is. A studious linguist, and aspiring diplomat, she is. Already worked together, I believe you and Pyakri have? One of our most senior militia officers, she is." He takes a pause, looking at Lobmab, almost as if waiting for some approval regarding his introductions of the others. She simply nods, and he moves on. "And... Elirom. A prominent Saryd Heliarch Arbiter who defected over a century ago. Been advising us since, he has." He waits for her to say something, before looking back at the table again and pointing to you. "Oh, of course. <first> <last>, the human, helped us greatly who has, and joining us today, who is."`
```

Elirom is a name, so should be capitalized.

Reply:
Yes, I know. I was using an automated script to do this and didn’t proofread it well enough apparently.